all/experiments/multiagent_env_experiment.py

```diff
@@ -29,7 +29,7 @@ def __init__(
             name=None,
             quiet=False,
             render=False,
-            save_freq=100,
+            save_freq=float('inf'),
```

Comment:
This looks left over from the other PR. Do you have strong opinions about merging that?

Reply:
No, I just branched off of my old branch instead of develop. Whoops. I can change this back.

Reply:
Reverted